gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -250,12 +254,15 @@ def __init__(self,
         exact data for variants and calls, without type matching.
       sample_name_encoding: specify how we want to encode sample_name mainly
         to deal with same sample_name used across multiple VCF files.
+      use_1_based_coordinate: specify whether the coordinates should be stored
+        in BQ needs on a 0 based exclusive (default) or 1 based inclusive basis.
```

Comment:
nit: `specify whether the coordinates should be stored in BQ **using** 0 based exclusive (default) or 1 based inclusive **coordinate**.`

Also please update below.